mtz-wizard-stepper.html

```diff
@@ -11,33 +15,219 @@
+        display: flex;
+        flex: 1;
+        align-items: center;
+        height: 72px;
+      }
+      /* Hover for non-disabled steps */
+      .step:not([disabled]):hover {
+        background-color: rgba(0,0,0,0.06);
+      }
+      /* Prevents clicking on steps */
+      .step[disabled] {
+        pointer-events: none;
+        z-index: 1;
+      }
+      /* Active/Completed Step */
+      .step[active] .badge,
+      .step[completed] .badge {
+        background-color: var(--primary-color, #2196F3); /* Blue 500 */
+      }
+      /* Make icons smaller except invalid */
+      .step:not([invalid]) iron-icon {
+        --iron-icon-height: 16px;
+        --iron-icon-width: 16px;
+      }
+
+
+      /* Positions badge and badge content */
+      .badge-wrapper {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        margin-right: 8px;
+        margin-left: 8px;
+        font-size: 12px;
+        color: #FFF;
+      }
+      .badge-wrapper,
+      .badge {
+        height: 24px;
+        width: 24px;
+      }
+      .badge {
+        background-color: rgba(0,0,0,0.38); /* 38% */
+        border-radius: 50%;
+      }
+      /* Position icon and counter over badge */
+      .badge-wrapper::before,
+      .badge-wrapper > iron-icon {
+        position: absolute;
+      }
+      /* Apply counter to step badge */
+      .step:not([completed]) > .badge-wrapper::before {
+        content: counter(step);
+      }
+      /* Badge should not display for invalid steps */
+      .step[invalid] .badge {
+        display: none;
+      }
+
+
+      .label-wrapper {
+        display: flex;
+        flex-direction: column;
+        margin-right: 8px;
+        overflow: hidden;
+      }
+      .label {
+        font-size: 14px;
+        color: rgba(0,0,0,0.38); /* 38% */
+      }
+      /* Ellipsis long text */
+      .label,
+      .optional {
+        white-space: nowrap;
+        text-overflow: ellipsis;
+        overflow: hidden;
+      }
+      .label + .optional {
+        font-size: 12px;
+        color: rgba(0,0,0,0.54); /* 54% */
+      }
+      /* Selected step */
+      .step[active] .label {
+        font-weight: 500; /* Medium */
+        color: rgba(0,0,0,0.87); /* 87% */
+      }
+
+
+      /* Apply invalid state */
+      .step[invalid] .label,
+      .step[invalid] .optional,
+      .step[invalid] iron-icon {
+        color: #F44336 !important;
+      }
+
+
+      /* Override browser specificity */
+      [hidden],
+      :host([hidden]) {
+        display: none !important;
       }
     </style>
 
-    <template is="dom-repeat" items="[[__steps]]">
-      <div on-tap="selectStep">
-        <div>
-          key: [[item.key]]
-        </div>
-        <div>
-          active: [[item.active]]
-        </div>
-        <div>
-          invalid: [[item.invalid]]
-        </div>
-        <div>
-          dirty: [[item.dirty]]
+    <template is="dom-repeat" items="[[__steps]]" as="step">
```

Comment:
For hybrid elements, don't templated areas require both the 2.x compatible and 1.x? I.e, here should <template is="dom-repeat"...> </> be encapsulated within a <dom-repeat> ...</> tag?